.jenkins/pipelines/manual-pr.Jenkinsfile

```diff
@@ -130,12 +130,16 @@ pipeline {
     post {
         always {
             script {
-                COMMITER_EMAILS.tokenize('\n').each {
```

Comment:
@rs-- Any thoughts on just removing the determine committer step and just email the person that triggered the build (or not at all)?

Reply:
I'm alright leaving it out. Personally, I tend to keep an eye on my builds anyways and miss the emails cause they go to my personal email address.